stock_by_warehouse_mrp_bom/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the bill of material's line to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: ./static/description/widget_by_warehouse.png
+    :alt: Example on Form View
+    :width: 600px
+
+Also, provide another widget in the product on the bill of material's line to see the quantity in stock by locations
```

Comment:
I think this can be omitted @luisg123v What do you think?

This function is from the `stock_by_warehouse` module

Reply:
@CarmenMiranda x2

I though that was requested by you.